app/controllers/user_journey_controller.rb

```diff
@@ -83,6 +97,10 @@ def find_certificate
     @certificate = Certificate.find_by_id(params[:certificate_id])
   end
 
+  def dual_running
```

Comment:
non blocking but `dual_running` returns a bool whereas `is_dual_running` does routing logic. It sorta feels like it this method should be called `is_dual_running?` and the one above should be `dual_running`. But feel free to ignore :)